### ISSUE TITLE
Advanced Blind doesn't work 

### ISSUE THREAD

**Issue by vzlgdu** [angeloxx/homebridge-myhome#9]
Hi Angelo, and congratulation for your great work!
I've a little bit complex myhome configuration and I'muisng with success your plugin (it works on RPI3).
I'm having problems with advanced Blinds (i've all F401 interfaces): The application homekit reports the correct % level of open/close status, but I'm not able to set my level from application, furthmore whe, from application, I push on blind icon, I can only open or close without the possibility to stop the motor...
Do you have some suggest to solve this?
Thanks

**Comment by angeloxx** [comment]
Hi, the blind widget doesn't have the "stop" command, but only the full up/full down and set a specific level of open status. Did you tried to set a specific %? Which kind and version of gateway do you have?

**Comment by vzlgdu** [comment]
Hi, the full up/down, works only with "MHBlind" widget; I would use "MHBlindAdvanced" widget because my interfaces are F401 and not F411. When I use "MHBlindAdvanced" configuration, the command doesn't work and the blind  doesn't move.

Please, let me know.

Thanks

**Comment by vzlgdu** [comment]
Update:
This are my configurations:
this never works (F401):
{"accessory" : "MHBlindAdvanced","name": "Persiana Studio","address": "0/0/13"},

this works only full/up/down (F401):
{"accessory" : "MHBlind","name": "Persiana Studio","address": "0/0/13"},

**Comment by angeloxx** [comment]
Can you answer to the second question?

**Comment by vzlgdu** [comment]
Modello F454
Versione FW2.0.50
Versione HW0.0.0
Versione Micro14.49.02
MAC address.....................
Versione kernel2.6.32
Versione del cancellatore d'eco 3207 - 1D09

**Comment by angeloxx** [comment]
I remember that F454 up to 2.0.46 does not support the F401 device so I suggest to use an MH202 or check with BTicino if the F401 advanced frames are supported by the firmware used by your F454. Tests are based on MH202 beta because advanced frames are not available when I developed this scrit.

**Comment by vzlgdu** [comment]
I use F454 web interface to set all by blinds to any position, so I think that it works with F401, anyway I restarted hombridge using MH202 as gateway but the result is the same...

**Comment by angeloxx** [comment]
Thank you; I'll check the issue the next time, I hope before the end of the year, and I'll update the ticket.

**Comment by vzlgdu** [comment]
ok. Thanks a lot. Bye

**Comment by angeloxx** [comment]
Here we go: I've tested MHBlindAdvanced with a MH201 gateway and it works; I don't know if commands are implemented and in which version of other MH Gateway (I suggest to check on myopencommunity and ask to Lorenzo or Giulia). I've tested the same configuration with F459 (an old and beta release) and it does not works. I'll try to get an MH202 or F455, but at the moment I can say that the frame is correct and it is supported by the "Hotel" gateway MH201.
